### PR TITLE
OTP demo

### PR DESCRIPTION
**Pull request by corinnekrych**
@edewit @danielpassos 
This is a fixed version of OTP demo (previous version in https://github.com/aerogear/aerogear-ios-cookbook/tree/1.6.x/Two-Factor was really wrong see https://issues.jboss.org/browse/AGIOS-390 for @abstractj comments on it).
This demo is based on Android demo for the UI flow. It uses iOS7 native QRCode capture (not based on zxing).

### CODE REVIEW

**Comment by abstractj** [comment]
@corinnekrych lovely! Will try this week!

**Comment by edewit** [comment]
tested and works :+1:

**Comment by danielpassos** [comment]
:+1:

File: Two-Factor/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Two-Factor
+------------
+Level: Beginner
+Technologies: Swift, iOS
+Summary: A basic example how to use OTP
```
Comment (danielpassos) [inline Two-Factor/README.md]: Markdown format problem

**Pull request merged**